pages/docs/prefetching.en-US.md

```diff
@@ -62,6 +62,25 @@ function App({ userId }) {
 }
 ```
 
+Inside React, you can call `preload` in effects.
```

Comment:
```suggestion
Within React rendering tree, you can call `preload` in effects.
```

I feel we should tweak the wording about `You can also preload it when hovering the button:` into a more general case: inside react tree

so we have 3 examples for different cases:

* general - preload outside of react
* preload in event callbacks
* preload in effects

Maybe the 2nd and 3rd one can be merged since they're both in react? then the tutorial can be easy and short to understand. just some of my personal thoughts

Reply:
Thank you! I'll update the section.